src/Task4.java

```diff
@@ -1,2 +1,17 @@
 public class Task4 {
+    //Створіть два масиви з 10 цілих чисел. Знайдіть квадратний корінь з кожного елемента в першому масиві та помістіть його результат у другий масив. Вивести другий масив на консоль.
+    public static void main(String[] args) {
+        int array1[];
+        array1 = new int[10];
+        float array2[];
+        array2 = new float[array1.length];
```

Comment:
Can be shorter:
int array1[] = new int[10];
float array2[] = new float[array1.length];